spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.